RELEASES.md

```diff
@@ -309,9 +309,9 @@ Misc
 
 Compatibility Notes
 -------------------
-- [`Command::before_exec` is now deprecated in favor of the
-  unsafe method `Command::pre_exec`.][58059]
-- [Use of `ATOMIC_{BOOL, ISIZE, USIZE}_INIT` is now deprecated.][57425] As you
+- [`Command::before_exec` is being replaced by the unsafe method
```

Comment:
```suggestion
- [`Command::before_exec` was replaced by the unsafe method
```

Replacement is not an ongoing process.

Reply:
Well in some sense it is, it goes on throughout the ecosystem during the future-deprecation period and beyond... but whatever, fine for me.^^

However, this would be the only use of past tense (comparing with other nearby items), that seems off then?

Reply:
It's probably fine as-is then :)